### ISSUE TITLE
Maximum width on YAML::Flow layout

### ISSUE THREAD

**Issue by oblitum** [jbeder/yaml-cpp#426]
Hi. As of now I've followed the tutorial and read some docs. I have a question/feature-request:

- Is there a way to limit width of emitter so that it automatically do line wrapping in Flow layout in case it gets to long?

For example, for the following YAML:

```yaml
foo: [
    a, b, c,
    d, e, f
]
```

Emitting it through the API I get:

```yaml
foo: [a, b, c, d, e, f]
```
The problem is when the sequence is quite long, I'd like to get some output like the former one, not too wide, not to tall.

**Comment by davidzchen** [comment]
This is a very old bug, but I also think it would be great to have this feature. If I understand correctly, pyYaml supports setting a line width: https://dpinte.wordpress.com/2008/10/31/pyaml-dump-option/

**Comment by davidzchen** [comment]
@jbeder Your thoughts on this?

**Comment by jbeder** [comment]
Sure, but I'm not sure how we'd get the output in the first example because it seems to know the entirety of the sequence, whereas the yaml-cpp API is streaming. But in principle yes.

**Comment by davidzchen** [comment]
Makes sense. For my case, it would not necessarily have to be in the format of the first comment. Simple line wrapping would be fine for me. Something like:

```
- '!Parameters': {DropTable: Normal, InitMotionStatus: 0, IsEnemyLiftable: true, IsPlayerPut: false,
    SharpWeaponJudgeType: 0}
```
